src/caduceus/outboundSender.go

```diff
@@ -412,6 +437,10 @@ func (obs *CaduceusOutboundSender) dispatcher() {
 		obs.queueDepthGauge.Add(-1.0)
 
 		obs.mutex.RLock()
+		urls := obs.urls
+		// Move to the next URL to try 1st the next time.
+		obs.urls = obs.urls.Next()
```

Comment:
I don't think this is needed.

Reply:
This allows us to move on to the next url & load balance across all alternatives.